### ISSUE TITLE
Last step- opening crx does not work

### ISSUE THREAD

**Issue by akh8zm** [mike-u/EditChromeThemes#7]
Hi! Thanks so much for the guide, it was very clear and helpful to a novice like myself. The only thing I wanted to comment on was that the last step, opening the `.crx` file in google chrome, did not work for me. Instead, I had to click the "Load unpacked" button within `chrome:\\extensions` developer mode, then choose my file directory that I had just packed. 
<img src="https://user-images.githubusercontent.com/56124630/88336148-7f719400-cd02-11ea-96d4-f2c9931b63f2.png">

**Comment by 2bbits-eddy** [comment]
For future reference.
It should be noted that your `.crx` file is your modified extension all packed together into one file that you can share.  You can drag and drop the `.crx` file into `chrome://extentions` page there will be a popup asking to add \*insert extension name*. It would be wise to also when changing parameters within the `manifest.json` file to also change the name of the extension, in the original post it is under the 5th line, `"name": "*insert extension name*"` to not get confused.

**Comment by mike-u** [comment]
You're right @akh8zm, I haven't seen that CRX_REQUIRED_PROOF_MISSING error before. It seems like it [started with Chrome 75](https://github.com/ahwayakchih/crx3#crx_required_proof_missing-chrome-and-chromium). @2bbits-eddy, thanks for figuring out that dragging it onto the window bypasses that, I'm not sure that's intentional but it's good to know there's an easy workaround.

Changing the manifest.json's `key` field to the contents of the generated .pem file and repacking didn't fix it. Same with changing `manifest_version` to 3 since that's the latest, and same with installing offline to avoid Google getting an [incorrect response from the `update_server` field](https://developer.chrome.com/docs/extensions/mv3/linux_hosting/#update_manifest). I think going forward I'll change the guide to only recommend loading unpacked extensions to avoid this whole mess.
